src/client/Microsoft.Identity.Client/AuthenticationResultMetadata.cs

```diff
@@ -24,20 +24,35 @@ public AuthenticationResultMetadata(TokenSource tokenSource)
         public TokenSource TokenSource { get; }
 
         /// <summary>
-        /// Total time (in ms) spent to service this request, in ms. Includes time spent making Http Requests <see cref="DurationInHttpInMs"/>, time spent
+        /// Time, in milliseconds, spent to service this request. Includes time spent making HTTP requests <see cref="DurationInHttpInMs"/>, time spent
         /// in token cache callbacks <see cref="DurationInCacheInMs"/>, time spent in MSAL and context switching.
         /// </summary>
         public long DurationTotalInMs { get; set; }
 
         /// <summary>
-        /// Time (in ms) MSAL spent in reading and writing to the token cache, i.e. in the OnBeforeAccess, OnAfterAccess etc. callbacks. 
+        /// Time, in milliseconds, MSAL spent during this request reading and writing to the token cache, i.e. in the OnBeforeAccess, OnAfterAccess, etc. callbacks. 
         /// Does not include internal MSAL logic for searching through the cache once loaded.
         /// </summary>
         public long DurationInCacheInMs { get; set; }
 
         /// <summary>
-        /// Time (in ms) MSAL spent for HTTP communication.
+        /// Time, in milliseconds, MSAL spent for HTTP communication during this request.
         /// </summary>
         public long DurationInHttpInMs { get; set; }
+
+        /// <summary>
+        /// Total tokens obtained by MSAL from the identity provider.
+        /// </summary>
+        public long TotalAccessTokensFromIdP { get; set; }
```

Comment:
I recommend that you start an API review as soon as possible, as we need to allow some time for reviewers.